rq/worker.py

```diff
@@ -696,7 +701,7 @@ def prepare_job_execution(self, job):
         with self.connection._pipeline() as pipeline:
             self.set_state(WorkerStatus.BUSY, pipeline=pipeline)
             self.set_current_job_id(job.id, pipeline=pipeline)
-            self.heartbeat(timeout, pipeline=pipeline)
+            self.heartbeat(pipeline=pipeline)
```

Comment:
When preparing to execute job, `heartbeat()` should be called with TTL of `DEFAULT_JOB_MONITORING_INTERVAL + 15`.